Imagine/Cache/CacheManager.php

```diff
@@ -138,10 +138,11 @@ public function getRuntimePath($path, array $runtimeConfig)
      * @param string $filter        The name of the imagine filter in effect
      * @param array  $runtimeConfig
      * @param string $resolver
+     * @param int    $referenceType The type of reference to be generated (one of the constants)
```

Comment:
lets say "(one of the UrlGenerator constants)" to be more specific about what we mean.